src/chains/ethereum_1/templates/chainActions/standard/token_transfer.ts

```diff
@@ -22,7 +22,6 @@ export const decomposeAction = (action: EthereumTransactionAction): ActionDecomp
     return {
       ...decomposed,
       args: {
-        ...decomposed.args,
```

Comment:
Removing this doesn't seem right to me. The revised code wont merge decompose.args with the new fields, it will simply replace args with the new values. Is there a reason you did this?

Reply:
The reason is we may have different interfaces between standart and chainSpesific implementation thus it wont replace but will add those properties in ethereum.